main/webservices/api/v2.php

```diff
@@ -291,6 +291,38 @@
             $restApi->setMessageRead($messageId);
             $restResponse->setData(['status' => true]);
             break;
+        case Rest::CREATE_SESSION_FROM_MODEL:
+            if (
+                empty($_POST['modelSessionId']) || empty($_POST['sessionName']) || empty($_POST['startDate']) || empty($_POST['endDate'])
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Line exceeds maximum limit of 120 characters; contains 137 characters](https://app.codacy.com/app/chamilo/chamilo-lms/pullRequest?prid=4554450)